.gitignore

```diff
@@ -5,3 +5,5 @@ result
 # Added by cargo
 
 /target
+
+.direnv/
```

Comment:
Missing final new line. Also we could safely remove all the other new lines.

Reply:
Thanks, the final newline has been added.